### PR TITLE
akita-removal main branch

### PR DESCRIPTION
**Pull request by phpgeekfr**

### CODE REVIEW

File: apps/crm/crm/src/app/crm/pages/dev-area/dev-area.component.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export class DevAreaComponent implements OnInit {
   ) { }
 
   async ngOnInit() {
-    this.token = await this.authService.user$.pipe(first())
+    this.token = await this.authService._user$.pipe(first())
```
[the remark this answers is not in the document]
Comment (phpgeekfr) [reply apps/crm/crm/src/app/crm/pages/dev-area/dev-area.component.ts:23]: Already done in sub-PR https://github.com/blockframes/blockframes/pull/8348

File: apps/backend-functions/src/negotiation.ts [hunk 2]
```diff
@@ -2,11 +2,34 @@ import { db } from './internals/firebase';
 import { Change, EventContext } from 'firebase-functions';
 import { createDocumentMeta, getDocument } from './data/internals';
 import { centralOrgId } from 'env/env.blockframes-ci';
-import { Contract, formatDocumentMetaFromFirestore, Negotiation, Organization, Sale, NotificationTypes, Offer, Timestamp } from '@blockframes/model';
+import { Contract, Negotiation, Organization, Sale, NotificationTypes, Offer, Timestamp, DocumentMeta } from '@blockframes/model';
 import { createNotification, triggerNotifications } from './notification';
 import { getReviewer, isInitial } from '@blockframes/contract/negotiation/utils'
 import { ContractStatus } from '@blockframes/model';
 
+function formatDocumentMetaFromFirestore( // TODO #8280 clean
```
Comment (GrandSchtroumpf) [inline apps/backend-functions/src/negotiation.ts:10]: Are you still using that ?

File: libs/user/src/lib/auth/+state/auth.service.ts [hunk 3]
```diff
@@ -252,7 +220,10 @@ export class AuthService extends FireAuthService<AuthState> {
     if (!intercom || !intercomId) return;
 
     const ref = doc(this.db, `users/${userCredential.user.uid}`) as DocumentReference<User>;
-    docData<User>(ref).pipe(take(1)).subscribe(user => {
+    fromRef(ref).pipe(
+      map(snap => snap.data()),
+      take(1)
+    ).subscribe(user => {
```
Comment (GrandSchtroumpf) [inline libs/user/src/lib/auth/+state/auth.service.ts:226]: You could do : 
```suggestion
    const user = await firstValueFrom(fromRef(ref).pipe(map(snap => snap.data()));
```

File: libs/user/src/lib/auth/pages/identity/identity.component.ts [hunk 4]
```diff
@@ -353,7 +353,7 @@ export class IdentityComponent implements OnInit, OnDestroy {
   }
 
   public async searchForInvitation() {
-    const { data: event } = await this.invitationService.getInvitationLinkedToEmail(this.form.get('email').value);
+    const event = await this.invitationService.getInvitationLinkedToEmail(this.form.get('email').value);
```
Comment (GrandSchtroumpf) [inline libs/user/src/lib/auth/pages/identity/identity.component.ts:356]:
```suggestion
    const email = this.form.get('email').value;
    const event = await this.invitationService.getInvitationLinkedToEmail(email);
```

**Review by GrandSchtroumpf — APPROVED** [review]

File: apps/backend-functions/src/negotiation.ts (hunk 2 above)
Comment (phpgeekfr) [reply apps/backend-functions/src/negotiation.ts:10]: moved in this file as it is only used here anymore, this will be reworked in next PR

**Pull request merged**